### DISCUSSION TITLE
Enable Addon without rebuilding network

### DISCUSSION THREAD

**Discussion by a-pir1** [hassio-addons/addon-zwavejs2mqtt#610] [Q&A] [answered]
I use Zwave JS and have already built a network on it before I found Zwave JS UI.  I have two questions:

1. Is there a way to enable this addon without having to delete my network and re-enroll every device?
2. Can I still add devices from the integrations UI like I used to, or now only from the Zwave JS UI?

**Comment by frenck** [comment, marked as answer]
> Is there a way to enable this addon without having to delete my network and re-enroll every device?

Sure, just switch. The network is not bound to add-ons or software. The network is on your Z-Wave controller (the stick).

> Can I still add devices from the integrations UI like I used to, or now only from the Zwave JS UI?

It provides the exact same functionality as the core add-on, but with the added bonus of another UI that provides extra options.

The difference is that this is a community add-on, and thus not supported by the Home Assistant project.
